srv/wordpress/wp-content/plugins/acore-wp-plugin/src/components/UserPanel/UserMenu.php

```diff
@@ -28,7 +28,8 @@ public static function I()
     function acore_user_menu()
     {
         if (Opts::I()->eluna_recruit_a_friend == '1') {
-            add_submenu_page('profile.php', 'Recruit a Friend', 'Recruit a Friend', 'subscriber', 'eluna-raf-progress', array($this, 'eluna_raf_progress_page'));
+            $user = wp_get_current_user();
```

Comment:
`$user` is it used?

Reply:
Nope, I just remove it, thanks for the notice.